apps/src/music/views/TimelineElement.jsx

```diff
@@ -44,7 +51,9 @@ const TimelineElement = ({
         width: barWidth * length - 4,
         height,
         top,
-        left
+        left,
+        ...(isInsideRandom && {borderColor: '#a1a1a1'}),
+        ...(isSkipping && {backgroundColor: '#808080'})
```

Comment:
minor, but could we put this in CSS to limit what inline styles are here? I wonder if we can reuse colors from `color.scss` or `tokens-dark.scss` too?

Reply:
I didn't find great matches in the existing files, but did move these into our own CSS.